Added bowtie2 tutorial and supporting images

triponfl opened the pull request.

Commit:
Added bowtie Tutorial and supporting data

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+# bowtie2 tutorial + a little bit about IVG
```
The name of the tool is IGV not IVG I believe (https://igv.org/)

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+# bowtie2 tutorial + a little bit about IVG
+written by: [trip - flavia tripon](https://github.com/triponfl)
+
+[~20 minutes] Bowtie2 aligns sequencing reads to reference sequences (so you will be able to use the output file to see coverage of a reference sequence if you add it into IVG). In relation to the VIRUSxDISCOVERY project, the aim is to determine reads from the raw SRA (SRR/ERR) file that align to the contig you've determined.
```
For the Tool Name use the \` character around the name like this, `bowtie2`

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+[~20 minutes] Bowtie2 aligns sequencing reads to reference sequences (so you will be able to use the output file to see coverage of a reference sequence if you add it into IVG). In relation to the VIRUSxDISCOVERY project, the aim is to determine reads from the raw SRA (SRR/ERR) file that align to the contig you've determined.
+
+**Tutorial Objective**: Align SRR11741344 reads (raw) to SRR11741344_2340 (the contig you found related to your virus)
```
Can you expand this to 2-3 sentences about "why" a reader would want to do this, you want to motivate the type of problem this tutorial is solving.

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+**Tutorial Objective**: Align SRR11741344 reads (raw) to SRR11741344_2340 (the contig you found related to your virus)
+
+## Input / Prerequisites
+**Required packages**: sra-tools, bowtie2
```
IGV is also used in this tutorial, add links to the tool websites at the end of the document.

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+official manual: [Tool Manual](https://github.com/BenLangmead/bowtie2/blob/master/MANUAL.markdown)
+
+example data to follow along: (my contig that you will see as SRR11741344_2340.fna in the screenshots)
+>SRR11741344_2340
```
Put the data inside a file into triple backticks so it creates blocktext \`\`\`

```
>SRR11741344_2340
GGGCCAGTTTCAGTCAACAAAACTGATTTCCTTATGGAAATTCCTTTCAGAAGGATGACTTTATGTAGTCAA
GGAGGAGGTGTTTCTCGTGAGGGAGAAACAGAGCTTTACAGTCAAGGCGTTGCTCACACCAGAGCAGGCGAA
GATTCTCACAAATCACGTGAAGTATCTCAACCGCGTGGATGACGTGACCAAGGGTGGTGCCTATGATGGTAT
...
```

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+### 1. installing bowtie2
+
+$ conda install bioconda::bowtie2
```
Put code into block code format

triponfl replied:
```
# Use conda to install bowtie2
conda install bioconda::bowtie2
```

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+Remember, that this is assuming that you already have sra-tools and an environment for this program.
+
+$ conda activate sra-env
```
block code

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+### 3. set your working directory to the folder containing your contig
+
+$ cd desktop/folder
```
block code

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+This is where sra-tools comes into play. Prefetch gets the file from ncbi and fasterq -dump splits it up.
+
+$ prefetch SRA#
```
block code

A "Variable Name" is defined as `$SRA` in sh script language, not as `SRA#`

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+$ prefetch SRA#
+
+$ fasterq -dump SRA#
```
```
fasterq -dump $SRA
```

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+Now that you've set up your raw SRA for bowtie2, you have to prepare your contig now as well.
+
+$ bowtie2-build your_contig some_new_name
```
block code

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+### 6. bowtie2 - aligning!
+
+$ bowtie2 -x some_new_name -1 SRA_fastq1 -2 SRA_fastq2 > your_output_name
```
block code and make the variable names `sh` compliant

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+### 1. samtools conversion (.sam -> .bam)
+
+$ samtools view -S -b your_output_name > your_output_name.bam
```
block code and make all three of these lines in the same block

ababaian commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+### Conclusion
+
+Hopefully this helps out!
```
:)

triponfl commented:
So sorry, I can't seem to find the comment in the PR? I'm not sure if I'm looking in the wrong place or I just had to generally go over the command lines I wrote and make the title clearer that it is a command line tutorial?

ababaian commented:
Comments can be found on the file: https://github.com/ababaian/VIRUSxDISCVRY/pull/92/files/59fdd1c186979f10cf8e4aaba69f9938036d03ea#diff-06c4d8829326dc4f5338fd3358d9db69eaaf2c2f767de7a922672ac560e2de33

triponfl commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+**Please note**: 
+
+- I operate on MacOs and use Conda to create separate environments (sra-env - you will see this below) and install various tools/programs. There are other ways to get sra-tools and bowtie2, but this tutorial starts with Conda.
+- If you're confused about this setup or wish to follow it, I recommend looking at: https://docs.anaconda.com/miniconda/miniconda-install/, https://bioconda.github.io/, https://anaconda.org/channels/bioconda/packages/sra-tools/overview, and creating the environment with this line: conda create -n name-of-your-environment -c conda-forge -c bioconda sra-tools.
```
`this is code` while this is text

```
this is block code

```

triponfl commented on tools/bowtie2.md:
```diff
@@ -0,0 +1,138 @@
+
+This is where sra-tools comes into play. Prefetch gets the file from ncbi and fasterq -dump splits it up.
+
+$ prefetch SRA#
```
```
SRA='SRR31337'

# Use prefetch to download binary compressed SRA data
prefetch $SRA

```

triponfl replied:
```
!#/bin/bash

# This is now a bash script

```

ababaian commented:
<img width="800" height="565" alt="image" src="https://github.com/user-attachments/assets/d6780c60-4256-4045-92fd-c31b1ff5e8ea" />
<img width="799" height="634" alt="image" src="https://github.com/user-attachments/assets/c78fa3de-6da5-4b53-85c0-b0dfeddb3c41" />

triponfl committed:
longer intro + block code addition + script

I'll go over the script I wrote out again because it doesn't seem to be working, but it might just have been my computer,

ababaian reviewed (approved):
:))
